Tasks/VsTest/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 69
+        "Patch": 70
```

Comment:
This should be a major version bump to 2.x since it's a rewrite and it should be marked preview.  For that reason, it shouldn't go in until next sprint 108.  Task locking and preview is coming in 107. 

This task is too critical to rollout without preview and an explicit hop from the customer.  This is in the default template.